index.bs

```diff
@@ -1236,6 +1247,12 @@ To <dfn noexport>create an attribution trigger</dfn> given a [=string=]
 1. Let |debugReportingEnabled| be false.
 1. If |value|["`debug_reporting`"] [=map/exists=] and is a [=boolean=], set
     |debugReportingEnabled| to value["`debug_reporting`"].
+1. Let |aggregationCoordinator| be null.
+1. If |value|["`aggregation_coordinator_identifier`"] [=map/exists=]:
+    1. If |value|["`aggregation_coordinator_identifier`"] is not a [=string=], return null.
+    1. If |value|["`aggregation_coordinator_identifier`"] is not an [=ASCII case-insensitive=] match
+        for any of [=aggregation coordinator=], return null.
```

Comment:
Are we sure we want to prevent source-registration entirely if the browser doesn't support a particular coordinator?

Reply:
It's trigger-registration. This was brought up in https://chromium-review.googlesource.com/c/chromium/src/+/4067585/comment/efd17f05_99960408 and per discussion, I think it's preferred to error out?

@johnivdel Any thoughts here? Thanks.

Reply:
If a particular coordinator is not supported by the browser, then even if it's encrypted using the default coordinator, it may not be decrypted properly. In this sense, probably it's fine to drop the trigger registration?